packages/expo-file-system/src/next/FileSystem.types.ts

```diff
@@ -95,4 +103,14 @@ export declare class File {
    * Creates a file.
    */
   create(): void;
+
+  /**
+   * Copies a directory.
```

Comment:
It's not a directory
```suggestion
   * Copies a file.
```